packages/connectivity/lib/connectivity.dart

```diff
@@ -56,6 +56,14 @@ class Connectivity {
     if (wifiName == '<unknown ssid>') wifiName = null;
     return wifiName;
   }
+
+  /// Obtains the wifi ip (IP) of the connected network
+  ///
+  /// Please note that it DOESN'T WORK on emulators (returns null).
+  Future<String> getWifiIP() async {
+    final String wifiIP = await _methodChannel.invokeMethod('wifiIPAddress');
+    return wifiIP;
```

Comment:
Just return directly `return await _methodChanel.invoke...`, no need for the variable..